### ISSUE TITLE
Can't start queue processing

### ISSUE THREAD

**Issue by gfcittolin** [netsmarttech/node-red-contrib-safe-queue#47]
I have a queue with "Start job at start?" *unchecked*, so that I have to start the queue manually. But after hitting the deploy button on Node-RED, I try to call "start-process", and then I get a `"Node out in process"` message in the debug tab, and therefore the queue processing doesn't start.

![image](https://user-images.githubusercontent.com/2856088/36313198-ff558cc8-1317-11e8-831c-73f41c426b36.png)

It starts only if I hit stop, and then start again.

Here is my flow

```
[{"id":"de6f7019.8185f","type":"tab","label":"Flow 1"},{"id":"480714f1.31370c","type":"file in","z":"de6f7019.8185f","name":"","filename":"json-events_122_2018-02-06.log","format":"lines","chunk":false,"sendError":false,"x":440,"y":140,"wires":[["45e20a29.e3caa4"]]},{"id":"620ec8a9.b00228","type":"inject","z":"de6f7019.8185f","name":"","topic":"","payload":"","payloadType":"str","repeat":"","crontab":"","once":false,"x":170,"y":140,"wires":[["480714f1.31370c"]]},{"id":"45e20a29.e3caa4","type":"queue in","z":"de6f7019.8185f","name":"","config":"3a96eacd.189176","sendError":true,"x":760,"y":140,"wires":[[]]},{"id":"3db24970.17e1c6","type":"queue control","z":"de6f7019.8185f","name":"","config":"3a96eacd.189176","operation":"stop-process","days":0,"x":350,"y":440,"wires":[[]]},{"id":"297008ec.a19f48","type":"inject","z":"de6f7019.8185f","name":"","topic":"","payload":"","payloadType":"str","repeat":"","crontab":"","once":true,"x":130,"y":440,"wires":[["3db24970.17e1c6"]]},{"id":"85d8cbb7.460548","type":"queue control","z":"de6f7019.8185f","name":"","config":"3a96eacd.189176","operation":"start-process","days":0,"x":350,"y":500,"wires":[[]]},{"id":"bbdf9a5f.6ebfd8","type":"inject","z":"de6f7019.8185f","name":"","topic":"","payload":"","payloadType":"str","repeat":"","crontab":"","once":false,"x":130,"y":500,"wires":[["85d8cbb7.460548"]]},{"id":"bb0975b7.473118","type":"queue control","z":"de6f7019.8185f","name":"","config":"3a96eacd.189176","operation":"queue-size","days":0,"x":350,"y":560,"wires":[[]]},{"id":"2166bd44.055892","type":"inject","z":"de6f7019.8185f","name":"","topic":"","payload":"","payloadType":"str","repeat":"2","crontab":"","once":false,"x":130,"y":560,"wires":[["bb0975b7.473118","eca174d7.e87f18","a5c1b956.8c2e08"]]},{"id":"eca174d7.e87f18","type":"queue control","z":"de6f7019.8185f","name":"","config":"3a96eacd.189176","operation":"done-size","days":0,"x":340,"y":620,"wires":[[]]},{"id":"a5c1b956.8c2e08","type":"queue control","z":"de6f7019.8185f","name":"","config":"3a96eacd.189176","operation":"error-size","days":0,"x":340,"y":680,"wires":[[]]},{"id":"49baf4cf.8a735c","type":"json","z":"de6f7019.8185f","name":"","pretty":false,"x":310,"y":240,"wires":[["bfe745e9.0b6be8"]]},{"id":"26665d1f.1603c2","type":"switch","z":"de6f7019.8185f","name":"","property":"payload.topic","propertyType":"msg","rules":[{"t":"regex","v":"^AG.*$","vt":"str","case":true},{"t":"else"}],"checkall":"true","outputs":2,"x":630,"y":240,"wires":[["de50126f.61b88"],["5c18df99.e2cf6"]]},{"id":"de50126f.61b88","type":"change","z":"de6f7019.8185f","name":"format influx","rules":[{"t":"set","p":"payload","pt":"msg","to":"payload.(\t    $topic := $split(topic, \"_\");\t    $loc := $topic[0];\t    $type := $topic[3];\t    $sa := $topic[4];\t    $pos := $topic[5];\t    \t    $tags := \"events,locale=\" & $loc & \",type=\" & $type & \",sa=\" & $sa;\t    $fields := \"position=\" & $pos & \",value=\" & payload;\t    \t    $tags & \" \" & $fields & \" \" & ts\t)","tot":"jsonata"}],"action":"","property":"","from":"","to":"","reg":false,"x":810,"y":220,"wires":[["f7fea7be.bf9088","e6f7a7f.67db958"]]},{"id":"bfe745e9.0b6be8","type":"function","z":"de6f7019.8185f","name":"timeToMillis","func":"if(msg && msg.payload && msg.payload.ts){\n    msg.payload.ts = new Date(msg.payload.ts).getTime()\n}\nreturn msg;","outputs":1,"noerr":0,"x":470,"y":240,"wires":[["26665d1f.1603c2"]]},{"id":"f7fea7be.bf9088","type":"http request","z":"de6f7019.8185f","name":"","method":"POST","ret":"txt","url":"http://127.0.0.1:8086/write?db=andon_events&precision=ms","tls":"","x":990,"y":240,"wires":[["a8fd00c2.c1f4d","7e7b580e.105198"]]},{"id":"1d2a9d9a.41c632","type":"queue out","z":"de6f7019.8185f","name":"","config":"3a96eacd.189176","x":120,"y":240,"wires":[["49baf4cf.8a735c","ce96dfa5.0a83"]]},{"id":"a8fd00c2.c1f4d","type":"queue ack","z":"de6f7019.8185f","name":"","config":"3a96eacd.189176","x":1210,"y":240,"wires":[]},{"id":"e6f7a7f.67db958","type":"debug","z":"de6f7019.8185f","name":"","active":false,"console":"false","complete":"true","x":970,"y":180,"wires":[]},{"id":"d746b374.66403","type":"queue control","z":"de6f7019.8185f","name":"","config":"3a96eacd.189176","operation":"resend-errors","days":0,"x":360,"y":760,"wires":[[]]},{"id":"e91b0764.b334c8","type":"inject","z":"de6f7019.8185f","name":"","topic":"","payload":"","payloadType":"str","repeat":"","crontab":"","once":false,"x":130,"y":760,"wires":[["d746b374.66403"]]},{"id":"ce96dfa5.0a83","type":"debug","z":"de6f7019.8185f","name":"","active":false,"console":"false","complete":"true","x":310,"y":280,"wires":[]},{"id":"7e7b580e.105198","type":"debug","z":"de6f7019.8185f","name":"","active":false,"console":"false","complete":"true","x":1150,"y":300,"wires":[]},{"id":"5c18df99.e2cf6","type":"change","z":"de6f7019.8185f","name":"format influx","rules":[{"t":"set","p":"payload","pt":"msg","to":"payload.(\t    \"stats,type=\" & topic & \" value=\" & payload & \" \" & ts\t)","tot":"jsonata"}],"action":"","property":"","from":"","to":"","reg":false,"x":810,"y":260,"wires":[["f7fea7be.bf9088","4464647f.671c8c"]]},{"id":"4464647f.671c8c","type":"debug","z":"de6f7019.8185f","name":"","active":false,"console":"false","complete":"true","x":970,"y":300,"wires":[]},{"id":"3a96eacd.189176","type":"queue config","z":"de6f7019.8185f","name":"","storage":"fs","path":"json_events","timeoutAck":"10000","startJob":false,"typeTimeout":"move-error","typeError":"move-error","retryTimeout":"","retryError":"","maxInMemory":"10000"}]
```

Can you check this out?

**Comment by igisho** [comment]
Same here with this log:
0|node-red | Error: spawn free EMFILE
0|node-red |     at exports._errnoException (util.js:1036:11)
0|node-red |     at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
0|node-red |     at onErrorNT (internal/child_process.js:359:16)
0|node-red |     at _combinedTickCallback (internal/process/next_tick.js:74:11)
0|node-red |     at process._tickDomainCallback (internal/process/next_tick.js:122:9)

**Comment by DiegoRampim** [comment]
Fix #49.
